Content.Shared/_RMC14/Areas/AreaSystem.cs

```diff
@@ -322,6 +339,12 @@ public override void Update(float frameTime)
                         {
                             areaGrid.Colors[pos] = minimapColor.Color;
                             found = true;
+
+                            if (areaGrid.Areas.TryGetValue(pos, out var areaProto))
+                            {
+                                if (HasComp<DamageableComponent>(anchored))
+                                    AddBuildableTile((ent,areaGrid), areaProto, anchored);
+                            }
```

Comment:
You shouldn't be doing this in the middle of the minimap trying to color itself. You should pull this out to its own section of code. Anchored here is relying on it having a minimap color component, which isn't correct.

Reply:
Done in [`b467dc2` (#6288)](https://github.com/RMC-14/RMC-14/pull/6288/commits/b467dc25bba4b47429a448cfd299a99ee52238a4)